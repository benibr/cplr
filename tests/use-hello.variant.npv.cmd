../bin/cplr 'puts("Hello.")' -npv
