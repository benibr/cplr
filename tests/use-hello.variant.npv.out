Generation phase
Generating code
Generating section main
Generated bytes: 182 code, 0 dump
Compiler condition:
main_0:1: warning: implicit declaration of function 'puts'
Compiler condition:
internal:159: error: ';' expected (got "<eof>")
Error: Compilation failed.
